### ISSUE TITLE
Different output directory name than kobaltBuild

### ISSUE THREAD

**Issue by binkley** [cbeust/kobalt#172]
A request.  Would you use a different output directory name than `kobaltBuild`?

Here's why.  Tab completion matches both `kobalt` and `kobaltBuild`, so using tab completion for the output directory takes extra typing.  I'm that lazy.  :-)

If I made a suggestion, just plain `build` would be fine, but I'm not too concerned as long as the initial characters are distinct from `kobalt`.

**Comment by cbeust** [comment]
I understand the laziness part :-)

build is not really an option since it’s already used by IDEA, though.
​

-- 
Cédric

**Comment by binkley** [comment]
Fair point on `build`.  It doesn't matter to me much what it is, as long as it starts with a distinct name I can use TAB completion with.
